extensions/data-transfer/portability-data-transfer-google/src/main/java/org/datatransferproject/datatransfer/google/media/GoogleMediaExporter.java

```diff
@@ -247,21 +253,31 @@ private ExportResult<MediaContainerResource> exportMediaContainer(
     for (PhotoModel photo : container.getPhotos()) {
       GoogleMediaItem photoMediaItem =
           getGoogleMediaItem(photo.getIdempotentId(), photo.getDataId(), photo.getName(), authData);
+
       if (photoMediaItem == null) {
         continue;
       }
 
-      photosBuilder.add(GoogleMediaItem.convertToPhotoModel(Optional.empty(), photoMediaItem));
+      try {
+        photosBuilder.add(GoogleMediaItem.convertToPhotoModel(Optional.empty(), photoMediaItem));
+      } catch(ParseException e) {
+        monitor.debug(() -> "parse exception occurred while converting photo, skipping this item");
```

Comment:
Let's make this 
```suggestion
        monitor.debug(() -> "Skipping GoogleMediaItem with id %s. Parse exception occurred while converting photo: %s", photoMediaItem.getIdempotentId(), e);
```
and make sure that for videos we say "converting video"